### ISSUE TITLE
expand-all doesn't expand screened comments

### ISSUE THREAD

**Issue by rahaeli** [dreamwidth/dw-free#714]
If you load an entry that has collapsed comments, and some of those comments are screened:
1. Using the "expand all" link in the comment control menu doesn't expand the screened comments.
2. Once "expand all" has been selected, and all the  comments have been expanded, there's no "expand" link on individual thread-starting comments, even if there are (screened) replies to that thread. (You have to expand each collapsed screened comment individually.)

Basically, if you have access to see screened comments, the "expand all" link should expand 'em.
